.github/workflows/neon_extra_builds.yml

```diff
@@ -269,6 +269,11 @@ jobs:
         password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
       options: --init
 
+    strategy:
+      fail-fast: false
+      matrix:
+        build_type: [ debug, release ]
+
```

Comment:
Are these changes necessary for the PR?

Reply:
this reduced the workflow time from 45 minutes to 25 minutes as these jobs now run in parallel.